doc/man3/X509_sign.pod

```diff
@@ -28,11 +32,16 @@ X509_sign() signs certificate B<x> using private key B<pkey> and message
 digest B<md> and sets the signature in B<x>. X509_sign_ctx() also signs
 certificate B<x> but uses the parameters contained in digest context B<ctx>.
 
-X509_verify() verifies the signature of certificate B<x> using public key
-B<pkey>. Only the signature is checked: no other checks (such as certificate
-chain validity) are performed.
+X509_verify_ex() verifies the signature of certificate B<x> using public key
+B<pkey>. Any cryptographic algorithms required for the verification are fetched
```

Comment:
`certificate I<a> .... public key I<r> ` and/or fix the prototypes to use more descriptive parameters.

Reply:
Fixed